packages/jest-leak-detector/src/index.ts

```diff
@@ -27,14 +27,14 @@ export default class {
 
     try {
       // eslint-disable-next-line import/no-extraneous-dependencies
-      weak = require('weak');
+      weak = require('weak-napi');
```

Comment:
you can move this to the top level now - it doesn't have to be conditional

Reply:
@SimenB I will move it to the top level when I add type for this package